ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/private/cucushift-installer-rehearse-azure-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-private
+  steps: 
+    pre:
+    - chain: ipi-conf-azure-sharednetwork
+    - ref: azure-bastion-pre
+    - chain: ipi-install
+    post:
+    - chain: gather-network
+    - chain: gather-core-dump
+    - chain: ipi-deprovision
+    - ref: ipi-deprovision-azure-resourcegroup
+    env:
+      CREATE_BASTION: "yes"
```

Comment:
azure-bastion-pre ref is already added in this workflow, I do not think the `CREATE_BASTION` is needed.